src/insane/ll_gbmbd.jl

```diff
@@ -24,7 +24,7 @@ Created 03 09 2020
 
 Returns the log-likelihood for a `iTgbmbd` according to `gbm-bd`.
 """
-function llik_gbm(psi::Vector{iTgbmbd}, 
+function llik_gbm(xi::Vector{iTgbmbd}, 
```

Comment:
oh ok, I understand since I had `psi` you changed it to `xi` rather than the letter. I guess to standardize lets have the greek letter always, I can do that anyways.